gulpfile.js

```diff
@@ -28,12 +28,13 @@ gulp.task('init', () => gulp.src("./typings.json").pipe(typings()));
 
 gulp.task('lint', ['tslint', 'eslint', 'depcheck']);
 
-gulp.task('build', () =>
-  mergeStream(
-    gulp.src('src/**/*.ts').pipe(typescript(tsProject)),
-    gulp.src(['src/**/*', '!src/**/*.ts'])
-  ).pipe(gulp.dest('lib'))
-);
+gulp.task('build', () => {
+  let tsReporter = typescript.reporter.defaultReporter();
```

Comment:
how far are we deviating from tools-common now? I suspect we need to do a re-sync...